base/range.jl

```diff
@@ -940,6 +940,11 @@ function ==(r::OrdinalRange, s::OrdinalRange)
     (first(r) == first(s)) & (step(r) == step(s)) & (last(r) == last(s))
 end
 
+==(r::AbstractUnitRange{<:Integer}, s::AbstractUnitRange{<:Integer}) =
```

Comment:
```suggestion
==(r::AbstractUnitRange, s::AbstractUnitRange) =
```
Is there a reason to restrict this to `Integer`s?

Reply:
I was unsure about omitting step comparison for different types, but yes comparing first and last is sufficient.